pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/test/java/org/apache/pinot/plugin/stream/kafka20/utils/MiniKafkaCluster.java

```diff
@@ -112,4 +115,9 @@ public void deleteTopic(String topicName)
       throws ExecutionException, InterruptedException {
     _adminClient.deleteTopics(Collections.singletonList(topicName)).all().get();
   }
+
+  public void deleteRecordsBeforeOffset(String topicName, int partitionId, long offset) {
+    _adminClient.deleteRecords(
```

Comment:
oo.. have never used this API. I am curious what happens when the offset provided here is invalid - does it throw or just ignores the call.

Reply:
not sure.. seems to be returning gracefully. this is only used in the tests

Reply:
ok